janitor/functions.py

```diff
@@ -4583,7 +4584,9 @@ def complete(
     else:
         df = df.reindex(sorted(reindex_columns)).reset_index()
 
-    if fill_value:
+    if fill_value is not None:
+        if not isinstance(fill_value, dict):
+            raise TypeError("""fill_value should be a dictionary.""")
```

Comment:
@ericmjl @samukweku 
Might be a silly question, but is there an intentional reason for triple quotes here?

Reply:
Now I can't remember but I think a member of the team suggested triple quotes. @loganthomas Is there any accepted way in terms of quotes?

Reply:
@samukweku 
I don't think there is an explicate rule here. I think it's a matter of preference. This certainly isn't *wrong*. Perhaps triple quotes are best for when long string messages are to be used in the error? This would allow us to write a long string without having to break lines and add strings in double quotes together. That's my hunch thought, I'm not an expert 😅. Perhaps @ericmjl has an idea here? Maybe short and sweet error messages use single-double quotes and anything beyond a single line uses triple-double quotes?

Consider these examples:

```python
###########################################
# From pyjanitor/scripts/check-autodoc.py #
###########################################
if len(missing_funcs) > 0:
    raise Exception(
        f"""The following functions have not yet been added: {missing_funcs}.
         Please add the function names to docs/reference/general_functions.rst
         under an appropriate section."""
    )

#####################################
# From pyjanitor/janitor/finance.py #
#####################################
def _check_currency(currency: str):
    if currency not in currency_set:
        raise ValueError(
            f"currency {currency} not in supported currency set, "
            f"{currency_set}"
        )


def _check_wb_country(country: str):
    if (country not in wb_country_dict.keys()) & (
        country not in wb_country_dict.values()  # noqa: PD011
    ):
        raise ValueError(
            f"country {country} not in supported World Bank country dict, "
            f"{wb_country_dict}"
        )


def _check_wb_years(year: int):
    if year < 1960:
        raise ValueError("year value must be 1960 or later")
```

Reply:
Agree with @loganthomas

[link from google dev](https://developers.google.com/edu/python/strings)
> A string literal can span multiple lines, but there must be a backslash \ at the end of each line to escape the newline. String literals inside triple quotes, """ or ''', can span multiple lines of text.

Reply:
Thanks @jk3587 and @loganthomas I'll fix it

Reply:
👍 Looks great!!